gui/src/components/Settings/Keybinds.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Box,
+  ListItem,
+  ListItemText,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { useState } from "react";
+
+import { useWallets } from "@/store";
+
+const MAX_CHANGE_WALLET_SHORTCUTS = 9;
```

Comment:
I had to redeclare the `MAX_CHANGE_WALLET_SHORTCUTS` constant twice due to linting issues.

Initially, when I created and exported the constant for use in the UI (`Sidebar.tsx`), the `yarn lint` failed due to import sorting. After fixing the sorting, the `yarn lint` passed, but running prettier caused a re-sorting, leading to a persistent loop. To resolve this, I opted for the current approach.

If you have any suggestions, please feel free to share.

Reply:
Update: this problem is already fixed.